frontend/src/views/Org/MembersPage/components/OrgGroupsTab/components/OrgGroupsSection/OrgGroupMembersModal.tsx

```diff
@@ -0,0 +1,163 @@
+import { useMemo,useState } from "react";
+import { faMagnifyingGlass,faUsers } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { createNotification } from "@app/components/notifications";
+import { OrgPermissionCan } from "@app/components/permissions";
+import {
+    Button,
+    EmptyState,
+    Input,
+    Modal,
+    ModalContent,
+    Table,
+    TableContainer,
+    TableSkeleton,
+    TBody,
+    Td,
+    Th,
+    THead,
+    Tr
+} from "@app/components/v2";
+import {
+    OrgPermissionActions,
+    OrgPermissionSubjects
+} from "@app/context";
+import { 
+    useAddUserToGroup,
+    useListGroupUsers,
+    useRemoveUserFromGroup} from "@app/hooks/api";
+import { UsePopUpState } from "@app/hooks/usePopUp";
+
+type Props = {
+  popUp: UsePopUpState<["groupMembers"]>;
+  handlePopUpToggle: (popUpName: keyof UsePopUpState<["groupMembers"]>, state?: boolean) => void;
+};
+
+export const OrgGroupMembersModal = ({
```

Comment:
More formatting issues. I will assume there'll be formatting issues spread out across the frontend. I think we should run the `prettier --write` command before merging this.

Reply:
Will run `prettier` in a separate PR.